### ISSUE TITLE
Super Mario Galaxy missing Textures

### ISSUE THREAD

**Issue by WilsontheWolf** [magcius/noclip.website#12]
In the Super Mario Galaxy (1) Comet Observatory there are missing textures related to the bedroom. Specifically, the spinner thing that unlocks it, the pipe connecting to it and the bedroom itself. 
![image](https://user-images.githubusercontent.com/33164598/53383780-fd771300-3935-11e9-8adc-75334ce87232.png)
![image](https://user-images.githubusercontent.com/33164598/53383824-154e9700-3936-11e9-9201-8ff7165ab927.png)
Other than that great porting 👍

**Comment by magcius** [comment]
Hmm, can you refresh again? I don't believe this happens consistently for me.

**Comment by WilsontheWolf** [comment]
yep here you go
![image](https://user-images.githubusercontent.com/33164598/53383912-62cb0400-3936-11e9-9b35-2af3d79807b0.png)

**Comment by WilsontheWolf** [comment]
I found a bunch of errors in the console
```
content.min.js:2 [Deprecation] Element.createShadowRoot is deprecated and will be removed in M73, around March 2019. Please use Element.attachShadow instead. See https://www.chromestatus.com/features/4507242028072960 for more details.
(anonymous) @ content.min.js:2
storage.googleapis.com/znoclip/z.noclip.website/j3d/smg/ObjectData/TicoGalaxy.arc:1 Failed to load resource: the server responded with a status of 404 ()
string_decoder.js:135 Could not fetch archive j3d/smg/ObjectData/TicoGalaxy.arc
(anonymous) @ string_decoder.js:135
storage.googleapis.com/znoclip/z.noclip.website/j3d/smg/ObjectData/SurpBeltConveyerExGalaxy.arc:1 Failed to load resource: the server responded with a status of 404 ()
string_decoder.js:135 Could not fetch archive j3d/smg/ObjectData/SurpBeltConveyerExGalaxy.arc
(anonymous) @ string_decoder.js:135
storage.googleapis.com/znoclip/z.noclip.website/j3d/smg/ObjectData/SurpSnowCapsuleGalaxy.arc:1 Failed to load resource: the server responded with a status of 404 ()
string_decoder.js:135 Could not fetch archive j3d/smg/ObjectData/SurpSnowCapsuleGalaxy.arc
(anonymous) @ string_decoder.js:135
storage.googleapis.com/znoclip/z.noclip.website/j3d/smg/ObjectData/SurpCocoonExGalaxy.arc:1 Failed to load resource: the server responded with a status of 404 ()
string_decoder.js:135 Could not fetch archive j3d/smg/ObjectData/SurpCocoonExGalaxy.arc
(anonymous) @ string_decoder.js:135
storage.googleapis.com/znoclip/z.noclip.website/j3d/smg/ObjectData/UFOKinokoUnderConstruction.arc:1 Failed to load resource: the server responded with a status of 404 ()
string_decoder.js:135 Could not fetch archive j3d/smg/ObjectData/UFOKinokoUnderConstruction.arc
(anonymous) @ string_decoder.js:135
storage.googleapis.com/znoclip/z.noclip.website/j3d/smg/ObjectData/SuperSpinDriverGreen.arc:1 Failed to load resource: the server responded with a status of 404 ()
string_decoder.js:135 Could not fetch archive j3d/smg/ObjectData/SuperSpinDriverGreen.arc
(anonymous) @ string_decoder.js:135
storage.googleapis.com/znoclip/z.noclip.website/j3d/smg/ObjectData/ButlerMap.arc:1 Failed to load resource: the server responded with a status of 404 ()
string_decoder.js:135 Could not fetch archive j3d/smg/ObjectData/ButlerMap.arc
(anonymous) @ string_decoder.js:135
storage.googleapis.com/znoclip/z.noclip.website/j3d/smg/ObjectData/AstroDomeEntranceBedroom.arc:1 Failed to load resource: the server responded with a status of 404 ()
string_decoder.js:135 Could not fetch archive j3d/smg/ObjectData/AstroDomeEntranceBedroom.arc
(anonymous) @ string_decoder.js:135
storage.googleapis.com/znoclip/z.noclip.website/j3d/smg/ObjectData/AstroStarPlateBedroom.arc:1 Failed to load resource: the server responded with a status of 404 ()
string_decoder.js:135 Could not fetch archive j3d/smg/ObjectData/AstroStarPlateBedroom.arc
(anonymous) @ string_decoder.js:135
storage.googleapis.com/znoclip/z.noclip.website/j3d/smg/ObjectData/MorphItemNeoFoo.arc:1 Failed to load resource: the server responded with a status of 404 ()
string_decoder.js:135 Could not fetch archive j3d/smg/ObjectData/MorphItemNeoFoo.arc
(anonymous) @ string_decoder.js:135
storage.googleapis.com/znoclip/z.noclip.website/j3d/smg/ObjectData/EffectTicoL.arc:1 Failed to load resource: the server responded with a status of 404 ()
string_decoder.js:135 Could not fetch archive j3d/smg/ObjectData/EffectTicoL.arc
(anonymous) @ string_decoder.js:135
storage.googleapis.com/znoclip/z.noclip.website/j3d/smg/ObjectData/EffectTicoS.arc:1 Failed to load resource: the server responded with a status of 404 ()
string_decoder.js:135 Could not fetch archive j3d/smg/ObjectData/EffectTicoS.arc
(anonymous) @ string_decoder.js:135
storage.googleapis.com/znoclip/z.noclip.website/j3d/smg/ObjectData/SurpSurfingLv2Galaxy.arc:1 Failed to load resource: the server responded with a status of 404 ()
string_decoder.js:135 Could not fetch archive j3d/smg/ObjectData/SurpSurfingLv2Galaxy.arc
(anonymous) @ string_decoder.js:135
storage.googleapis.com/znoclip/z.noclip.website/j3d/smg/ObjectData/SurpCubeBubbleExLv2Galaxy.arc:1 Failed to load resource: the server responded with a status of 404 ()
string_decoder.js:135 Could not fetch archive j3d/smg/ObjectData/SurpCubeBubbleExLv2Galaxy.arc
(anonymous) @ string_decoder.js:135
storage.googleapis.com/znoclip/z.noclip.website/j3d/smg/ObjectData/CutBushGroup.arc:1 Failed to load resource: the server responded with a status of 404 ()
string_decoder.js:135 Could not fetch archive j3d/smg/ObjectData/CutBushGroup.arc
(anonymous) @ string_decoder.js:135
storage.googleapis.com/znoclip/z.noclip.website/j3d/smg/ObjectData/SurpTamakoroExLv2Galaxy.arc:1 Failed to load resource: the server responded with a status of 404 ()
string_decoder.js:135 Could not fetch archive j3d/smg/ObjectData/SurpTamakoroExLv2Galaxy.arc
(anonymous) @ string_decoder.js:135
storage.googleapis.com/znoclip/z.noclip.website/j3d/smg/ObjectData/AstroOverlookObj.arc:1 Failed to load resource: the server responded with a status of 404 ()
string_decoder.js:135 Could not fetch archive j3d/smg/ObjectData/AstroOverlookObj.arc
(anonymous) @ string_decoder.js:135
storage.googleapis.com/znoclip/z.noclip.website/j3d/smg/ObjectData/TicoStarRing.arc:1 Failed to load resource: the server responded with a status of 404 ()
string_decoder.js:135 Could not fetch archive j3d/smg/ObjectData/TicoStarRing.arc
(anonymous) @ string_decoder.js:135
storage.googleapis.com/znoclip/z.noclip.website/j3d/smg/ObjectData/AstroTorchLightRed.arc:1 Failed to load resource: the server responded with a status of 404 ()
string_decoder.js:135 Could not fetch archive j3d/smg/ObjectData/AstroTorchLightRed.arc
(anonymous) @ string_decoder.js:135
storage.googleapis.com/znoclip/z.noclip.website/j3d/smg/ObjectData/TicoComet.arc:1 Failed to load resource: the server responded with a status of 404 ()
string_decoder.js:135 Could not fetch archive j3d/smg/ObjectData/TicoComet.arc
(anonymous) @ string_decoder.js:135
storage.googleapis.com/znoclip/z.noclip.website/j3d/smg/ObjectData/SurpTearDropGalaxy.arc:1 Failed to load resource: the server responded with a status of 404 ()
string_decoder.js:135 Could not fetch archive j3d/smg/ObjectData/SurpTearDropGalaxy.arc
(anonymous) @ string_decoder.js:135
storage.googleapis.com/znoclip/z.noclip.website/j3d/smg/ObjectData/ButlerExplain.arc:1 Failed to load resource: the server responded with a status of 404 ()
string_decoder.js:135 Could not fetch archive j3d/smg/ObjectData/ButlerExplain.arc
(anonymous) @ string_decoder.js:135
storage.googleapis.com/znoclip/z.noclip.website/j3d/smg/ObjectData/KinopioAstro.arc:1 Failed to load resource: the server responded with a status of 404 ()
string_decoder.js:135 Could not fetch archive j3d/smg/ObjectData/KinopioAstro.arc
(anonymous) @ string_decoder.js:135
storage.googleapis.com/znoclip/z.noclip.website/j3d/smg/ObjectData/SurpTeresaMario2DGalaxy.arc:1 Failed to load resource: the server responded with a status of 404 ()
string_decoder.js:135 Could not fetch archive j3d/smg/ObjectData/SurpTeresaMario2DGalaxy.arc
(anonymous) @ string_decoder.js:135
storage.googleapis.com/znoclip/z.noclip.website/j3d/smg/ObjectData/SuperSpinDriverPink.arc:1 Failed to load resource: the server responded with a status of 404 ()
string_decoder.js:135 Could not fetch archive j3d/smg/ObjectData/SuperSpinDriverPink.arc
(anonymous) @ string_decoder.js:135
storage.googleapis.com/znoclip/z.noclip.website/j3d/smg/ObjectData/SurpPeachCastleFinalGalaxy.arc:1 Failed to load resource: the server responded with a status of 404 ()
string_decoder.js:135 Could not fetch archive j3d/smg/ObjectData/SurpPeachCastleFinalGalaxy.arc
(anonymous) @ string_decoder.js:135
storage.googleapis.com/znoclip/z.noclip.website/j3d/smg/ObjectData/SurpFishTunnelGalaxy.arc:1 Failed to load resource: the server responded with a status of 404 ()
string_decoder.js:135 Could not fetch archive j3d/smg/ObjectData/SurpFishTunnelGalaxy.arc
(anonymous) @ string_decoder.js:135
storage.googleapis.com/znoclip/z.noclip.website/j3d/smg/ObjectData/StarReturnDemoStarter.arc:1 Failed to load resource: the server responded with a status of 404 ()
string_decoder.js:135 Could not fetch archive j3d/smg/ObjectData/StarReturnDemoStarter.arc
(anonymous) @ string_decoder.js:135
storage.googleapis.com/znoclip/z.noclip.website/j3d/smg/ObjectData/BenefitItemOneUp.arc:1 Failed to load resource: the server responded with a status of 404 ()
string_decoder.js:135 Could not fetch archive j3d/smg/ObjectData/BenefitItemOneUp.arc
(anonymous) @ string_decoder.js:135
storage.googleapis.com/znoclip/z.noclip.website/j3d/smg/ObjectData/GrandStarReturnDemoStarter.arc:1 Failed to load resource: the server responded with a status of 404 ()
string_decoder.js:135 Could not fetch archive j3d/smg/ObjectData/GrandStarReturnDemoStarter.arc
(anonymous) @ string_decoder.js:135
storage.googleapis.com/znoclip/z.noclip.website/j3d/smg/ObjectData/SurpTransformationExGalaxy.arc:1 Failed to load resource: the server responded with a status of 404 ()
string_decoder.js:135 Could not fetch archive j3d/smg/ObjectData/SurpTransformationExGalaxy.arc
(anonymous) @ string_decoder.js:135
storage.googleapis.com/znoclip/z.noclip.website/j3d/smg/ObjectData/BrightSun.arc:1 Failed to load resource: the server responded with a status of 404 ()
string_decoder.js:135 Could not fetch archive j3d/smg/ObjectData/BrightSun.arc
```

**Comment by magcius** [comment]
Fixed in 10e9a7f55a69a5622f27b90de36646f4b96f0403.

**Comment by WilsontheWolf** [comment]
Works thanks 👍
